doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -33,48 +35,91 @@ Generated keys default to retaining the seed used.
 The seed is also by default retained when keys are loaded from B<PKCS#8> files
 in the seed format.
 When available, the seed parameter is also used during key export and import,
-with keys regenerated from the seed even if separately provided on import.
+with keys regenerated from the seed even when also provided on import.
```

Comment:
This section is currently a point of contention.

Reply:
Adding a reference to the provider config section.